config/tls.ini

```diff
@@ -1,5 +1,6 @@
 ; See 'haraka -h tls'
 
+[inbound]
```

Comment:
Won't this change the following settings paths? IE, instead of `main.key`, they'll be `inbound.key`, and thus break them?

Reply:
Previously my *whole* tls.ini looked like this:

```
ciphers=...
```

Now I am running this:
```
[inbound]
ciphers=...
[outbound]
ciphers=DEFAULT
```
Default keys are still loaded from same place at least in my case. I looked at tls_socket.js and it seems that section name has no influence to paths also